chat-plugins/uno.js

```diff
@@ -488,7 +488,7 @@ class UNOgame extends Rooms.RoomGame {
 	}
 
 	onWin(player) {
-		this.sendToRoom(`|raw|<div class="broadcast-green">Congratulations to ${SG.nameColor(player.name, true, true)} for winning the game of UNO!</div>`, true);
+		this.sendToRoom(`|raw|<div class="broadcast-green">Congratulations to ${(player.name, true, true)} for winning the game of UNO!</div>`, true);
```

Comment:
This is incorrect, it would error / output something wrong. Do `${player.name}` instead, the `()`s and the true, true were part of the SG.nameColor function.